base/stat.jl

```diff
@@ -14,7 +14,7 @@ export
     isfile,
     islink,
     ismount,
-    ispath,
+    nz,
```

Comment:
???

Reply:
... I guess I mashed  the keyboard with my face...